src/components/v5/common/ActionSidebar/partials/forms/EditColonyDetailsForm/partials/SocialLinksTable/SocialLinksTable.tsx

```diff
@@ -19,10 +20,10 @@ const SocialLinksTable: FC<SocialLinksTableProps> = ({ name }) => {
   const isMobile = useMobile();
   const fieldArrayMethods = useFieldArray({ name });
   const value = useWatch({ name });
-  const getMenuProps = useGetTableMenuProps(
-    fieldArrayMethods,
-    setSocialLinkIndex,
-  );
+  const { readonly } = useAdditionalFormOptionsContext();
+  const getMenuProps = readonly
+    ? () => undefined
+    : getTableMenuProps(fieldArrayMethods, setSocialLinkIndex, readonly);
```

Comment:
Since you've got a condition dependent on `readonly`, there's most likely no point in passing it to `getTableMenuProps` - it's always `false`.